### ISSUE TITLE
Productive != Effective

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#651]
# _Context_

A person in our community who is struggling to get into a sustainable work/life routine that matches their [Circadian Clock](https://en.wikipedia.org/wiki/Circadian_clock) mentioned that they feel more "productive" in the evenings 20:00 - 01:00 which makes it difficult for them to "switch off" at 22:00. 

The person in question is _fixated_ on the idea/word "productive" and I _keep_ trying to _explain_ to them that "productive" is not the measure they should use when reflecting on how they use their time.

## What is "_Productivity_" and How does it _Differ_ from "_Effectiveness_"?

As an organisation, we need to be _clear_ on the difference between "**productive**" and "***effective***". 

"Productive" means you are churning out a lot of "stuff" regardless of importance or priority. 
"Effective" means you're focussed and getting the most important, valuable & strategic things done. 🥇 

As Dan North @tastapod said in his _great_ talk on "**Patterns of _Effective_ Teams**": 
> "_Productive teams is a **massive red herring**_."
> Productive comes from the latin "producere" ... 
> "The ability to to make lots of a thing".
> "If I have a more productive system, it produces more stuff.
> The goal of software development is not to produce stuff.
> The goal of software development is to impact a business in some way, hopefully positively.
> We are not about writing software, we're about solving business problems.
> "_**Effective** teams manage to solve gnarly business problems with tiny tiny bits of software._
> _If you can solve gnarly business problems with **no** software, you win at software_."
> I do not want productive teams, I do not want productive engineers, 
> What I want is people who are really really good at understanding the heart of the problem
> and just solving for that. I'll pay a lot of money for that.
[![image](https://user-images.githubusercontent.com/194400/51906983-70d04800-23bd-11e9-96d1-36ab150da804.png)](https://youtu.be/lvs7VEsQzKY?t=177)
https://youtu.be/lvs7VEsQzKY?t=177
Seriously, _watch_ the talk. It's a _much_ better use of your time than anything on Netflix.

## I would _much_ rather have an _effective **hour**_ than a _productive **day**_. 

_Volume_ of code or other creative work done is only relevant when _practicing_.
When you are engaged in deliberate practice, you should still strive for effectiveness,
(_i.e. don't write the same "**hello world**" app ten thousand times and expect to magically be recruited to work at Google! build something that you - or someone else - want/need. then **iterate & refine**._) 
[_quantity_](https://www.inc.com/ilan-mochari/stephen-king-on-quantity-versus-quality.html) is a good metric for _beginners_, because quality only comes _after_ doing a [_lot_](https://twitter.com/jack/status/411515505120268288) of creative work.

When _solving_ a specific problem the objective is not to "produce" _more_, 
rather it's to have the mental clarity and focus to do _exactly_ what _needs_ to be done
to solve the problem without any [_mistakes_](https://en.wikipedia.org/wiki/Retained_surgical_instruments) causing any harm in the process.

If you are even _slightly_ tired, you should not perform highly demanding cognitive tasks.
(_this is fairly logical to some people but it's worth repeating/writing for others ..._) 
Both your cognitive, decision-making and judgement abilities are gradually "spent" throughout the day. 
If you wake up at 08:00 and spend the whole day (_e.g: 08:30 - 19:30_) working at a screen
you have been awake for 12 hours (_working for 11h_) and your brain is has been [_fried_](https://youtu.be/OvM5QawiqJg) for the day.
Then are your _claim_ to be "**most _productive_**" from 20:00 - 01:00 ...?
You might _feel_ like you are "getting things done" at this time but the _reality_ is both your brain and body are exhausted and there is a _high_ likelihood that 
(a) you will be _distracted_ and not being _effective_ with your evening.
(b) you are _cognitively impaired_ because you are _exhausted_.
(c) you _think_ you are "making progress" but all you're doing is running very fast on a treadmill.

## I _Get_ It, This is The _Only_ Time You _Can_ Do Your _Own_ Work ...

I've been where the person in question is (_in life_) where you feel that there are so many distractions, interruptions, meetings and demands on your time during the "work day" (_e.g: 09:00 - 18:00 or whatever your "on the clock" time is_) that you feel you can _only_ get your _own_ work done after everyone **`else`** has left (_you alone_) for the day; I _get_ it.

What I am trying to highlight with this issue/post is that you might _feel_ that 20:00 - 01:00 is the _only_ time you can work, but the reality is _very_ different. If you _measure_ your _effectiveness_ by tracking _what_ you **_actually_ achieve** during the day (_rather than how much "**activity**" you appear to have_) you will rapidly see that your evenings, when you are exhausted and unfocussed, are nowhere _near_ as ***effective*** as spending the _first_ hour of the day on the most _important_ task. 

# “_Don't mistake activity for achievement_.” <br />~ [John Wooden](https://www.ted.com/talks/john_wooden_on_the_difference_between_winning_and_success)

Your wheels may appear to be turning, but is your vehicle going in the right direction?
Would you get to your destination faster if you took removed the park/hand break and were rested enough to not make any mistakes along the way?

## Chronotype?

There is evidence for the concept of ["**chronotypes**"](https://en.wikipedia.org/wiki/Chronotype). _Research_ suggests that adolescents and young adults prefer a delayed sleep period and _can_ have a more "optimal functioning time" if their day starts later. see: 
+ Morningness-Eveningness, Chronotypes and Health-Impairing Behaviors in Adolescents https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818690/
+ Identifying the Best Times for Cognitive Functioning Using New Methods: Matching University Times to Undergraduate Chronotypes https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5395635
+ Chronotype and academic achievement among online learning students https://www.sciencedirect.com/science/article/pii/S1041608013001799
+ Chronotype variation drives night-time sentinel-like behaviour in hunter–gatherers
https://royalsocietypublishing.org/doi/10.1098/rspb.2017.0967
+ http://www.cambridgecognition.com/blog/entry/how-your-body-clock-may-affect-cognition

The research tends to focus on people between the ages of 17 and 21 and concludes that Highschool and University Lectures should start later in the day for certain people. Nowhere in the literature does it _suggest_ that a young adult should work 10 hours at a desk and only _after_ that time they will achieve their highest level of _focus_. There is **_zero_ evidence** that studying (_or doing any other cognitively demanding task_) at the _end_ of one's day is the most _effective_ action. 

Research has proposed an evolutionary link between chronotype and nighttime vigilance in ancestral societies. https://royalsocietypublishing.org/doi/10.1098/rspb.2017.0967
But this is _not_ an _excuse_ to stay up late because "_nighttime vigilance_" of a hunter–gatherer has virtually _nothing_ to do with creative or other cognitively demanding work. It makes _sense_ that our paleolithic ancestors would have designated one or more members of the tribe to "keep watch" for predators and rival tribes at night and these individuals may have _adapted_ to being awake at night.
The same is true of modern-day "night shift" workers, they (_partially_) adapt to working at night and sleeping during the day.  But there is a _big_ difference between staying _alert_ and observing for intruders or doing low cognitive intensity tasks as a [_shift_ worker](https://en.wikipedia.org/wiki/Shift_work) compared to the work of a creative technologist that requires intense focus and creativity. 

# Bottom Line 

_Track_ your Tasks/Time, Monitor your _effectiveness_ not "productivity" and be _honest_ with yourself when you analyse the **data**; one _fresh_, _focussed_ & _effective_ hour in the morning without any distractions or interruptions to get your most important task done is worth 6 hours of evening  "productivity" when you are tired, half-distracted & mind buzzing from the hectic day.

Go to bed when you _first_ feel tired in the evening in accordance with your circadian rhythm. 
Wake/get up [_early_](https://github.com/dwyl/sleep/issues/32) (_e.g: before the sun rises!_) and get your _most_ important task done _first_ thing. 
The _momentum_ this "_first win_" of the day will give you will set you up to succeed for the rest of the day!

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/80465290-e5809200-8932-11ea-830c-38d89e8fc6d7.png)

**Comment by nelsonic** [comment]
Want to Be More Productive? Try Doing Less.
https://hbr.org/2020/05/want-to-be-more-productive-try-doing-less
https://news.ycombinator.com/item?id=23284912
